### PR TITLE
Extend API with submission image, speaker avatar and fix speaker biography

### PR DESCRIPTION
**Pull request by sreimers**
Extend API with submission image, speaker avatar and fix speaker biography

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] My change is listed in the CHANGELOG.rst if appropriate.

### CODE REVIEW

File: src/pretalx/api/serializers/speaker.py [hunk 1]
```diff
@@ -12,19 +12,20 @@ class SubmitterSerializer(ModelSerializer):
 
     def get_biography(self, obj):
         if self.context.get('request') and self.context['request'].event:
-            getattr(obj.profiles.filter(event=self.context['request'].event).first(), 'biography', '')
+            return getattr(obj.profiles.filter(event=self.context['request'].event).first(), 'biography', '')
```
Comment (rixx) [inline src/pretalx/api/serializers/speaker.py]: Good catch!

**Comment by rixx** [comment]
Meh, GitHub reviews are weird.

What I meant to say was this: I'm not sure how having an "avatar.png" is helpful – we should return the full URL to the image, I think, so that API consumers can actually use the information provided.

**Comment by sreimers** [comment]
Yes, the api already provides the full image/avatar url. a better documention hint would be maybe "https://example.net/media/avatar.png"

**Comment by rixx** [comment]
The API does that if you use the `speakers/` endpoint, but for the `speakers` field inside the `submissions/` endpoint, it just shows "avatar.png". Could you have a look at that?

**Comment by sreimers** [comment]
Ah, sure i review this.

**Comment by sreimers** [comment]
ok done

**Comment by rixx** [comment]
Thank you very much!

**Pull request merged**